### ISSUE TITLE
timeline generation failure

### ISSUE THREAD

**Issue by ace-han** [vuepress-reco/vuepress-theme-reco#209]
- [ ] I confirm that this is an issue rather than a question.

#### What is actually happening?

No timeline generated by following the instruction from [official doc](https://vuepress-theme-reco.recoluan.com/en/views/1.x/timeline.html)

#### Other relevant information

However, adding Front Matter `categories` to each article, timeline will show as expected.

So I think it worth mentioning in [official doc](https://vuepress-theme-reco.recoluan.com/en/views/1.x/timeline.html)

Failure demo links:
https://github.com/hand-in-hand-edu/hand-in-hand-edu
https://hand-in-hand-edu.github.io/hand-in-hand-edu/timeline/

- Your OS: macOS Catalina 10.15.5
- Node.js version: v12.16.1
- VuePress version: 1.3.1
- VuePress-theme-reco version: 1.4.7
- Which package manager did you use for the install(npm/cnpm/yarn)? npm

**Comment by recoluan** [comment]
Hello, brother.

The `timeline` page is automatically generated and there is no need to add a file locally, so just delete the `timeline` folder and the files inside it from your project.

**Comment by ace-han** [comment]
Hi, man

I just forgot pushing my last change.

It shows `Yesterday Once More!` but no timeline generation

By adding Front Matter `categories` to each article timeline will show as expected. You could folk https://github.com/hand-in-hand-edu/hand-in-hand-edu to try it out

**Comment by recoluan** [comment]
The situation you mentioned is normal now.

When working with data, vuepress-theme-reco works like this:

- There are two kinds of articles, one is a blog, the other is a document. Blog posts typically contain authors, dates, categories, and so on, which documents do not need.

- So when working with blog data, vuepress-theme-reco skips posts that don't have categories, and the timeline data skips posts that don't have time on top of it.

- For documentation related articles, vuepress-theme-rec tend to put them together and not in a blog, such as the custom sidebar of Vuepress.

Vuepress-theme-reco makes a conscious distinction between these two types of data as a convention.

When I saw your question today, I found that I did not mention this specification in the document, which is my problem.I'm going to refine the documentation later.

I hope my answer was helpful to you.
